src/halmos/config.py

```diff
@@ -88,7 +88,7 @@ class Config:
 
     config: str = arg(
         help="path to the configuration file",
-        global_default=None,
+        global_default=os.path.join(os.getcwd(), "halmos.toml"),
```

Comment:
if i understand correctly, this default value is only used when generating the help message, as `resolve_config_files()` doesn't use this default value, right?

in that case, here can we use a meta variable, say `ROOT`, instead of `os.getcwd()`, to highlight that the config file will be searched under the root, which is not necessarily the current directory?

Reply:
it now says:

```
options:
  -h, --help            show this help message and exit
  --root ROOT           project root directory (default: current working directory)
  --config FILE         path to the config file (default: ROOT/halmos.toml)
```

Reply:
> this default value is only used when generating the help message

yeah that's right